package.json

```diff
@@ -55,11 +55,15 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router": "^6.0.0",
+    "react-router-dom": "^6.3.0",
```

Comment:
This should also be included in `peerDependencies` (>=6.0.0)

Reply:
done